src/main/java/seedu/address/logic/commands/exceptions/HistoryException.java

```diff
@@ -0,0 +1,14 @@
+package seedu.address.logic.commands.exceptions;
+
+public class HistoryException extends CommandException {
```

Comment:
Why is a new HistoryException needed? Is CommandException not enough?
(Also extending off from CommandException is playing a very dangerous game, since catch CommandException will still catch this)